### PR TITLE
Updated terms conditions

### PR DESCRIPTION
**Pull request by borjainglefernandez**
## Status

- Done

## GitHub Issues addressed

- N/A

## What I did

Updated the terms of conditions:
- Added clause to "User Conduct" that users should not name those outside of the faculty listed on the website without explicit consent.
- Revised the date the terms were last reviewed
- Revised the state the laws of the terms apply to (from FL to VA)
- Changed "." to ";" to match the existing pattern

## Tests

- Looked at it locally.

## Questions/Discussions/Notes

- Does the wording make sense?

## Merging the PR

- Who should merge this PR?
  - [] I will merge it myself
  - [] The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - [] preserve the history
  - [] squash-merge

### CODE REVIEW

**Comment by borjainglefernandez** [comment]
pylint is failing because of `index.py`, which I did not change. This is porbabbly because pylint updated to look for `open without explicitly specifying an encoding`. Should I add the fix to this pr?

**Comment by alex-shen1** [comment]
I'd say go ahead and fix it here - this is gonna be the soonest PR merged in, so we might as well knock it out here and not later.

File: tcf_website/templates/about/terms.html [hunk 1]
```diff
@@ -71,8 +72,8 @@ <h4>User Conduct</h4>
...
-        <li>use or attempt to use another's account, service or system without authorization from us, or create a false identity on the Service or the Site.</li>
-        <li>upload, post, transmit, share, store or otherwise make available content that, in our sole judgment , is objectionable or which restricts or inhibits any other person from using or enjoying the Site, or which may expose us or our users to any harm or liability of any type.</li>
+        <li>use or attempt to use another's account, service or system without authorization from us, or create a false identity on the Service or the Site;</li>
+        <li>upload, post, transmit, share, store or otherwise make available content that, in our sole judgment , is objectionable or which restricts or inhibits any other person from using or enjoying the Site, or which may expose us or our users to any harm or liability of any type;</li>
```
Comment (alex-shen1) [inline tcf_website/templates/about/terms.html:76]: Not your change, but there's an empty space after `sole judgement` - might as well fix it while you fix Pylint?

**Comment by borjainglefernandez** [comment]
Should be good to go now

**Review by alex-shen1 — APPROVED** [review]

**Pull request merged**